src/util/apolloWrapper.js

```diff
@@ -1,22 +1,45 @@
 import compose from "recompose/compose";
 import branch from "recompose/branch";
+import lifecycle from "recompose/lifecycle";
 
+import renderQueue from "util/renderQueue";
 
-const renderNullIfNotLoaded = branch(
-    props => props.data.loading,
-    () => () => null
-);
+const updateQueueOnChange = lifecycle({
+    componentDidMount() {
```

Comment:
componentWillMount seems appropriate. I changed componentDidUpdate to componentWillUpdate too for consistency.